### PR TITLE
Add tranx id to events

### LINKED ISSUE #234: Add TranxID to Events
Proto changes for vegaprotocol/vega/#4598

### PR DESCRIPTION
**Pull request by ettec**
Closes #234 Add tranx id to events

see /vegaprotocol/vega/#4598

### CODE REVIEW

**Review by pscott31 — APPROVED** [review]
A slightly weird abbreviation of transaction :)

File: sources/vega/events/v1/events.proto [hunk 1]
```diff
@@ -507,4 +507,5 @@ message BusEvent {
   // Version of bus event
   uint32 version = 4;
   string chain_id = 5;
+  string tranx_id = 6;
```
Comment (jeremyletang) [inline sources/vega/events/v1/events.proto:510]: Can we just rename it transaction_hash?

Comment (jeremyletang) [reply sources/vega/events/v1/events.proto:510]: No need for the abreviation really

Comment (ettec) [reply sources/vega/events/v1/events.proto]: sure, done

**Pull request merged**